### PR TITLE
Use hardcoded language (migrated)

### PR DESCRIPTION
**Pull request by sofietoft**
Updating https://github.com/umbraco/UmbracoDocs/pull/4293 for the migrated docs.

@dampee , @kjac - please have a look and let me know if I did it correctly based on the comments 🙏

### CODE REVIEW

**Review by dampee — APPROVED** [review]

File: umbraco-cms/reference/notifications/editormodel-notifications/customizing-the-links-box.md [hunk 1]
```diff
@@ -9,14 +9,25 @@ For a content item, Umbraco will show a **Links** box within the **Info** conten
...
 ```C#
 public void Handle(SendingContentNotification notification)
 {
     notification.Content.Urls = new[]
     {
-        new UrlInfo($"/products/?id={notification.Content.Id}", true, CultureInfo.CurrentCulture.Name)
+        new UrlInfo($"/products/?id={notification.Content.Id}", true, null)
+    };
+}
+
```
Comment (kjac) [inline umbraco-cms/reference/notifications/editormodel-notifications/customizing-the-links-box.md:22]:
```suggestion
```

```

Comment (kjac) [reply umbraco-cms/reference/notifications/editormodel-notifications/customizing-the-links-box.md:22]: Should end the code block before the "specify culture" comment below

File: umbraco-cms/reference/notifications/editormodel-notifications/customizing-the-links-box.md [hunk 2]
```diff
@@ -9,14 +9,25 @@ For a content item, Umbraco will show a **Links** box within the **Info** conten
...
+It is also possible to specify the culture:
+
```
Comment (kjac) [inline umbraco-cms/reference/notifications/editormodel-notifications/customizing-the-links-box.md:24]: Add "```C#" to start the next code block

File: umbraco-cms/reference/notifications/editormodel-notifications/customizing-the-links-box.md [hunk 3]
```diff
@@ -9,14 +9,25 @@ For a content item, Umbraco will show a **Links** box within the **Info** conten
...
 public void Handle(SendingContentNotification notification)
 {
     notification.Content.Urls = new[]
     {
-        new UrlInfo($"/products/?id={notification.Content.Id}", true, CultureInfo.CurrentCulture.Name)
+        new UrlInfo($"/products/?id={notification.Content.Id}", true, null)
+    };
+}
+
+It is also possible to specify the culture:
```
Comment (kjac) [inline umbraco-cms/reference/notifications/editormodel-notifications/customizing-the-links-box.md:23]:
```suggestion
If the content item has multiple cultures, we can specify the link culture like this: 
```

Comment (kjac) [reply umbraco-cms/reference/notifications/editormodel-notifications/customizing-the-links-box.md:23]: If this rephrase makes more sense, the phrase _"or remove the box entirely by providing an empty list of links:"_ later in this document should also be updated to something like _"We can also remove the Links box entirely by providing an empty list of links:"_

**Comment by kjac** [comment]
@sofietoft left a few comments - nothing major, feel free to disregard them

File: umbraco-cms/reference/notifications/editormodel-notifications/customizing-the-links-box.md (hunk 3 above)
Comment (sofietoft) [reply umbraco-cms/reference/notifications/editormodel-notifications/customizing-the-links-box.md]: Where exactly are you proposing to add this rephrasing? 😇

**Pull request merged**